light_curves/code/gaia_functions.py

```diff
@@ -477,3 +357,55 @@ def Gaia_plot_lightcurves(df_lc , nbr_objects):
     plt.show()
     
     return(True)
+def Gaia_extract_median_photometry(gaia_table):
```

Comment:
@afaisst are you ok with the plan to remove the median photometry from the light curve plots?

Reply:
After discussion with Andreas, the code now keeps the median photometry function in case a future user wants to use it.  Possible uses are to compare with similar bands of other photometry, combining them into a light curve.  This sort of cross calibration is beyond the scope of this demo notebook, but we will leave the function there in case someone else wants to do it.